j2735_v2x_msgs/CMakeLists.txt

```diff
@@ -19,55 +19,97 @@ project(j2735_v2x_msgs)
 set(CMAKE_CXX_STANDARD 17)
 
 find_package(carma_cmake_common REQUIRED)
-carma_check_ros_version(2)
 carma_package()
 
-## Find ament macros and libraries
-find_package(rosidl_default_generators REQUIRED)
-find_package(ament_cmake REQUIRED)
-find_package(ament_cmake_auto REQUIRED)
-find_package(std_msgs REQUIRED)
-
-# generate messages
-file(GLOB msg_files msg/*.msg)
-set(msgs )
-foreach(filePath ${msg_files})
-  get_filename_component(name ${filePath} NAME)
-  set(msgs ${msgs} msg/${name})
-endforeach(filePath)
-
-rosidl_generate_interfaces(
-    ${PROJECT_NAME}
-        ${msgs}
+find_package(ros_environment REQUIRED)
+
+set(ROS_VERSION $ENV{ROS_VERSION})
+
+if(${ROS_VERSION} EQUAL 1) # ROS 1
+  find_package(catkin REQUIRED
+    COMPONENTS
+    message_generation
+    std_msgs
```

Comment:
ROS1 Cmake file had geometry_msgs, is that actually not needed? same comment on `generate_messages` and `catkin_package`

Reply:
Yes none of the messages in the package use `geometry_msgs`, so this package shouldn't be needed